com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -456,6 +458,7 @@ public HDRenderPipeline(HDRenderPipelineAsset asset)
                     supportStreaming = m_Asset.currentPlatformRenderPipelineSettings.supportProbeVolumeStreaming
                 });
                 RegisterRetrieveOfProbeVolumeExtraDataAction();
+                SupportedRenderingFeatures.active.overridesLightProbeSystemWarningMessage = "This Light Probe system is not active because the pipeline uses Probe Volumes and the systems cannot co-exist.\nTo disable Probe Volumes make sure the feature is disabled in the lighting section of the active HDRP Asset.";
```

Comment:
what about using the open button here to scroll to the hdrp asset ? HDEditorUtils.QualitySettingsHelpBox
![image](https://user-images.githubusercontent.com/61282776/154329040-2e91c0c4-370f-4fba-8061-4f177f2592bd.png)

For the button on the main apv window that's more tricky because it should be pipeline agnostic i guess

Reply:
The problem is that this message is triggered via trunk and needs to be a string.

Reply:
That kind of thing we should def. put in the Probe volume message when feature is disabled tho, for a next PR!